paasta_tools/mesos/master.py

```diff
@@ -76,12 +81,13 @@ def fetch(self, url, **kwargs):
     def _file_resolver(self, cfg):
         return self.resolve(open(cfg[6:], "r+").read().strip())
 
+    @retry(KazooTimeoutError, tries=5, delay=0.5, logger=logger)
     def _zookeeper_resolver(self, cfg):
         hosts, path = cfg[5:].split("/", 1)
         path = "/" + path
 
         retry = KazooRetry(max_tries=10)
-        with zookeeper.client(hosts=hosts, read_only=True, connection_retry=retry) as zk:
+        with zookeeper.client(hosts=hosts, read_only=True, connection_retry=retry, command_retry=retry) as zk:
```

Comment:
I don't really understand what these args are for. Isn't the decorator enough on its own?